packages/modules/pricing/integration-tests/__tests__/services/pricing-module/calculate-price.spec.ts

```diff
@@ -40,7 +40,7 @@ const createPriceLists = async (
   rules: object = defaultRules,
   prices = defaultPriceListPrices
 ) => {
-  return await service.createPriceLists([
+  return service.createPriceLists([
```

Comment:
nit: This intentional?

Reply:
Nope. By mistake, but it shouldn't change the flow or the behavior of the code. However, happy to revert it back